exporter/instanaexporter/factory_test.go

```diff
@@ -59,6 +59,29 @@ func TestLoadConfig(t *testing.T) {
 
 		err = cfg.Validate()
 
+		require.NoError(t, err)
+		assert.Equal(t, &Config{
+			ExporterSettings: config.NewExporterSettings(config.NewComponentID(typeStr)),
+			HTTPClientSettings: confighttp.HTTPClientSettings{
+				Endpoint:        "http://example.com/api/",
+				Timeout:         30 * time.Second,
+				Headers:         map[string]string{},
+				WriteBufferSize: 512 * 1024,
+			},
+			Endpoint: "http://example.com/api/",
+			AgentKey: "key1",
+			CAFile:   "ca.crt",
+		}, cfg)
+	})
+
```

Comment:
This test should remain as it is. Then there should be a test with a ca file because ca file is optional.